### PR TITLE
i18n: spanish translation (closed due to errors)

### PR DESCRIPTION
**Pull request by EA4HAM**
#### What does this PR do and why is it necessary?
This Pull Request contains the translation of welle.io GUI to the Spanish language which is spoken about 592million people.
#### How was it tested? How can it be tested by the reviewer?
It has been read 3 times in order to avoid translation error. The syntax of the XML file has been tested at https://www.w3schools.com/xml/xml_validator.asp
#### Any background context you want to provide?
I use the welle.io software to listen to the DAB radio and experimenting with RTL dongles as a HAM radio operator (my callsign is EA4HAM) and I noticed that there is no Spanish translation of the GUI, so I decided to translate it to that language.

### CODE REVIEW

File: src/welle-gui/i18n/es_ES.ts [hunk 1]
```diff
@@ -0,0 +1,1766 @@
...
+        <source>Ring buffer size (roughly): </source>
+        <translation>Tamaño de búfer de anillo</translation>
+    </message>
+</context>
+<context>
+    <name>ServiceDetails</name>
+    <message>
+        <location filename="../QML/expertviews/ServiceDetails.qml" line="10"/>
+        <source>Service Details</source>
+        <translation>Dtalles de la estación</translation>
```
Comment (andimik) [inline src/welle-gui/i18n/es_ES.ts:1501]: Typo?

File: src/welle-gui/i18n/es_ES.ts [hunk 2]
```diff
@@ -0,0 +1,1766 @@
...
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="244"/>
+        <source>Frisian</source>
+        <translation>Frisón</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="245"/>
+        <source>Irish</source>
+        <translation>Irlanés</translation>
```
Comment (andimik) [inline src/welle-gui/i18n/es_ES.ts:525]: Typo? But I do not speak Spanish

**Comment by tenzap** [comment]
Please add the .qm file in addition to the .ts file.
IIRC, `lupdate` can produce it.

File: src/welle-gui/i18n/es_ES.ts [hunk 3]
```diff
@@ -0,0 +1,1766 @@
...
+    </message>
+    <message>
+        <location filename="../radio_controller.cpp" line="152"/>
+        <source>Error while opening file </source>
+        <translation>Error al abrir el fichero </translation>
+    </message>
+    <message>
+        <location filename="../radio_controller.cpp" line="255"/>
+        <source>Playback failed</source>
+        <translation>Reprdoucción fallida</translation>
```
[the remark this answers is not in the document]
Comment (EA4HAM) [reply src/welle-gui/i18n/es_ES.ts]: Hi Andreas,

Yes, it is a typo. It should be "Reproducción fallida". File updated.

Regards,
Nándor

File: src/welle-gui/i18n/es_ES.ts (hunk 1 above)
Comment (EA4HAM) [reply src/welle-gui/i18n/es_ES.ts]: Hi Andreas,

Yes, it is a typo. It should be "Detalles de la estación". File updated.

Regards,
Nándor

File: src/welle-gui/i18n/es_ES.ts (hunk 2 above)
Comment (EA4HAM) [reply src/welle-gui/i18n/es_ES.ts]: Hi Andreas,

Yes, it is a typo. It should be "Irlandés". File updated.

Regards,
Nándor

**Comment by EA4HAM** [comment]
> Please add the .qm file in addition to the .ts file. IIRC, `lupdate` can produce it.

Hi,

I corrected the typo errors indicate by Andreas.

I managed to create the .qm file
`U:\pruebas\qt>lrelease welle.io\src\welle-gui\i18n\es_ES.ts
Updating 'welle.io\src\welle-gui\i18n\es_ES.qm'...
    Generated 330 translation(s) (330 finished and 0 unfinished)`

I did a commit to the file src/welle-gui/welle-gui.pro but I managed to revert it.

Sorry for the mistakes, this is my very first time using GitHub and I'm not familiar with how it works.

Regards,
Nándor

**Comment by EA4HAM** [comment]
Closing this PR due to errors
- typo erros (corrected)
- missing .qm archive
- unnecessary commits (to the file welle-gui.pro)

**Pull request closed without merging**